gems/smithy-client/lib/smithy-client/plugins/protocols.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Client
+    module Plugins
+      # @api private
+      class Protocols < Plugin
```

Comment:
Should this be just `Protocol` instead of plural? Given that a single protocol is used at a time.